pyServer/manifests/linux/workloadbackup

```diff
@@ -5,21 +5,18 @@ copy,/etc/hostname
 copy,/etc/waagent.conf
 copy,/var/lib/waagent/*.xml
 copy,/var/spool/cron/tabs/root
+copy,/opt/msawb/etc*
 echo,
 
 echo,### Gathering Log Files ###
 copy,/var/log/waagent*
 copy,/var/log/dmesg*
 copy,/var/log/syslog*
 copy,/var/log/azure/*
-copy,/var/opt/msawb/catalog/InquiryCatalog/*/*.bin
-copy,/var/opt/msawb/catalogRegisteredObjectInfoCatalog/*/*.bin
-copy,/var/opt/msawb/catalog/WorkloadExtDatasourceCatalog/*/*.bin
-copy,/var/opt/msawb/catalog/WorkloadSchedules/*/*.bin
-copy,/var/opt/msawb/catalog/SyncObjectsCatalog/DatasourceSyncTable/*.bin
-copy,/var/opt/msawb/catalog/SyncObjectsCatalog/AlertEventsTable/*.bin
+copy,/opt/msawb/var/lib*
```

Comment:
(applies to all three entries)
I am not sure that this will do what you hope it will.  Does this folder have many _files_ that start with "lib"?  Perhaps you can run the commands below on a sample VM and send me the `~/tree_output.log file` offline?

```
sudo apt-get install tree
tree -d /opt/msawb/etc > ~/tree_output.log
tree -d /opt/msawb/var/lib >> ~/tree_output.log
tree -d /opt/msawb/var/log >> ~/tree_output.log
```